pkg/sidecar/docker_reactor.go

```diff
@@ -79,6 +80,12 @@ func (d *DockerReactor) ResolveServices(runid string) {
 		os.Getenv(EnvInfluxdbHost),
 	}
 
+	additionalHosts := strings.Split(os.Getenv(EnvAdditionalHosts), ",")
+	logging.S().Infow("additional hosts", "hosts", os.Getenv(EnvAdditionalHosts))
+	if len(additionalHosts) != 1 || additionalHosts[0] != "" {
+		wantedRoutes = append(wantedRoutes, additionalHosts...)
+	}
```

Comment:
Can't we just do the following here?
```suggestion
	wantedRoutes = append(wantedRoutes, additionalHosts...)
```

Reply:
Yep, that's a good idea!